datasafe-simple-adapter/datasafe-simple-adapter-impl/src/test/java/de/adorsys/datasafe/simple/adapter/impl/SimpleDatasafeAdapterTest.java

```diff
@@ -300,6 +305,40 @@ void testTwoUsers(WithStorageProvider.StorageDescriptor descriptor) {
 
     }
 
+    @ParameterizedTest
+    @MethodSource("minioOnly")
+    @SneakyThrows
+    void testExhaustedInputStream(WithStorageProvider.StorageDescriptor descriptor) {
+        myinit(descriptor);
+        mystart();
+        long openFileHandlesBefore = 0;
+        OperatingSystemMXBean os = ManagementFactory.getOperatingSystemMXBean();
+        if(os instanceof UnixOperatingSystemMXBean){
+            openFileHandlesBefore = ((UnixOperatingSystemMXBean) os).getOpenFileDescriptorCount();
```

Comment:
What happens if this runs in parallel ? Descriptor count can change while test is running

Reply:
yes fixed now with limited connections.